packages/math-input/src/components/input/math-input.tsx

```diff
@@ -626,6 +667,33 @@ class MathInput extends React.Component<Props, State> {
         }
     };
 
+    // We want to allow the user to be able to focus the input via click
+    // when using ChromeOS third-party browsers that use mobile user agents,
+    // but don't actually simulate touch events.
+    handleClick: (arg1: React.MouseEvent<HTMLDivElement>) => void = (e) => {
+        e.stopPropagation();
+
+        // Hide the cursor handle on touch start, if the handle itself isn't
+        // handling the touch event.
+        this._hideCursorHandle();
+
+        // Cache the container bounds, so as to avoid re-computing. If we don't
+        // have any content, then it's not necessary, since the cursor can't be
+        // moved anyway.
```

Comment:
Do we still want to cache things? I know we talked about maybe not needing that.